### PR TITLE
add chloromethane substitution reactions and fix nomenclature

### PR DESCRIPTION
**Pull request by sleepbot-zapp**

### CODE REVIEW

**Comment by CodeWithSwastik** [comment]
Please follow the contribution guidelines. Use lowercase and IUPAC names of the compounds.

**Comment by sleepbot-zapp** [comment]
done

File: reactions.json [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
   },
-  "propane": {
+  "Propane": {
     "Cl/uv": "2-chloropropane",
     "Br/uv": "2-bromopropane"
+  },
+  "Chloromethane": {
+      "NaOH (aq.)": "methanol",
+      "KOH (aq.)": "methanol",
+      "AgCN": "methyl carbylamine",
```
Comment (sachin-sankar) [inline reactions.json:17]: Methyl carbylamine is a common name. 
Iupac name should be
**isocyanatomethane**

File: reactions.json [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
-    "Cl/uv": "1-chloromethane",
-    "Br/uv": "1-bromomethane"
+  "Methane": {
+    "Cl/uv": "chloromethane",
+    "Br/uv": "bromomethane"
   },
-  "ethane": {
-    "Cl/uv": "1-chloroethane",
-    "Br/uv": "1-bromoethane"
+  "Ethane": {
```
Comment (sachin-sankar) [inline reactions.json:6]: 1-chloroethane and 1-bromoethane is preferred IUPAC name

File: reactions.json [hunk 3]
```diff
@@ -1,14 +1,22 @@
 {
-  "methane": {
-    "Cl/uv": "1-chloromethane",
-    "Br/uv": "1-bromomethane"
+  "Methane": {
+    "Cl/uv": "chloromethane",
```
Comment (sachin-sankar) [inline reactions.json:3]: 1-chloroethane and 1-bromoethane are preferred IUPAC name

File: reactions.json [hunk 4]
```diff
@@ -1,14 +1,22 @@
...
-  "propane": {
+  "Propane": {
     "Cl/uv": "2-chloropropane",
     "Br/uv": "2-bromopropane"
+  },
+  "Chloromethane": {
+      "NaOH (aq.)": "methanol",
+      "KOH (aq.)": "methanol",
+      "AgCN": "methyl carbylamine",
+      "KCN": "ethanenitrile",
```
Comment (sachin-sankar) [inline reactions.json:18]: **acetobnitrile** is preferred IUPAC name

**Review by sachin-sankar — CHANGES_REQUESTED** [review]
Errors with names are there. I would suggest changing them and making a new pull request.

**Comment by CodeWithSwastik** [comment]
I'll edit it myself

**Comment by sachin-sankar** [comment]
Ok

**Pull request merged**